dependencies.tsv

```diff
@@ -107,3 +107,4 @@ gopkg.in/natefinch/npipe.v2	git	c1b8fa8bdccecb0b8db834ee0b92fdbcfa606dd6	2016-06
 gopkg.in/retry.v1	git	01631078ef2fdce601e38cfe5f527fab24c9a6d2	2017-05-31T09:12:38Z
 gopkg.in/tomb.v1	git	dd632973f1e7218eb1089048e0798ec9ae7dceb8	2014-10-24T13:56:13Z
 gopkg.in/yaml.v2	git	1be3d31502d6eabc0dd7ce5b0daab022e14a5538	2017-07-12T05:45:46Z
+k8s.io/client-go	git	e121606b0d09b2e1c467183ee46217fa85a6b672	2017-02-13T17:46:50Z
```

Comment:
Is that actually enough? Aren't there tons of other dependencies? (the things that client-go depends on)

You can confirm by creating a fresh, temporary GOPATH and trying to build Juju in there.